src/bin/pgcopydb/pgcmd.h

```diff
@@ -183,6 +183,41 @@ typedef struct ArchiveContentArray
 } ArchiveContentArray;
 
 
+/*
+ * Section options to use with pg_dump and pg_restore.
+ */
+typedef enum
+{
+	PRE_DATA = 0,
+	POST_DATA,
+} SectionOption;
```

Comment:
Please add a prefix such as PG_RESTORE_SECTION_ to these constants. C lacks namespaces :/

Reply:
Or better yet, can you just re-use PG_DUMP_SECTION_PRE_DATA etc maybe?

Reply:
makes sense :thumbsup: